### ISSUE TITLE
Minor styling issue on error messages in Site Kit Admin bar 

### ISSUE THREAD

**Issue by wpdarren** [google/site-kit-wp#6369]
## Bug Description

I noticed that if I go offline while the stats are loading on the Site Kit admin bar, the styling on the `'You are probably offline'` error message could be improved so that the `'Retry didn't work?`" text is more aligned with the retry button, or, is displayed underneath.  Only a minor issue but thought it was worth raising. 

![image.png](https://images.zenhubusercontent.com/5f99965247ef5a1da6985d7d/c29ad313-ef30-4199-8994-9de5be66ae80)

## Steps to reproduce

1. Hover over the Site Kit admin bar on the front end of a website. 
2. Go offline temporarily
3. Observe the error message 'You are probably offline' 

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

- Styles for report error actions should be made available to the Admin Bar.
- The report error actions should be allowed to wrap so that the overlapping elements appear underneath.

## Implementation Brief

* Add styles to: `.googlesitekit-report-error-actions` in https://github.com/google/site-kit-wp/blob/develop/assets/sass/components/global/_googlesitekit-error-text.scss:

```
.googlesitekit-report-error-actions {
    align-items: center;
    display: flex;
    gap: 1em;
}
```

### Test Coverage

* No new tests are needed; VRTs might need updating.

## QA Brief

* Open a page that has analytics data on the frontend
* Temporary go offline (by disabling wifi module or unplugging lan cable)
* Hover on Site Kit from admin top bar menu
* Verify that on larger screens 1920px message is rendering next to the retry button with a proper spacing between them
* Verify that on smaller screens 1440px or less when modules are narrow - message renders bellow the retry button
* Go back to Site Kit dashboard while offline
* Verify that existing modules which should show error (full width over the long module) - render the message inline with retry button with proper spacing between
* Basically when there is enough space message will render next to the retry button, on screens where modules are tighter together and narow, message should render bellow the retry button

## Changelog entry

* Improve styling of error messages in the Site Kit Admin Bar.

**Comment by aaemnnosttv** [comment]
The styling here has changed a bit since this was reported (the message is irrelevant):

<img width="558" alt="image" src="https://user-images.githubusercontent.com/1621608/219479319-ad848bf4-e5ed-496c-9db7-dbef4cb82660.png">

I found this is cleaned up by adding a few styles

```css
.googlesitekit-report-error-actions {
    display: flex;
    gap: 1em;
    align-items: center;
}
```

<img width="555" alt="image" src="https://user-images.githubusercontent.com/1621608/219479442-38554691-08ab-4576-8d1f-13d0327a532d.png">

**Comment by tofumatt** [comment]
ACs here are good, and I think @aaemnnosttv's suggestion works as a simple IB too so I've added it 🙂

**Comment by eugene-manuilov** [comment]
IB ✔️

**Comment by mohitwp** [comment]
### QA Update ✅ 

Tested on dev environment.
Verified on mobile, tablet and desktop viewports.
Now error messages under side kit admin bar appearing with proper space and as per AC and QAB


<details>

**Latest :**

![image](https://github.com/google/site-kit-wp/assets/94359491/35afc28b-b2ff-493c-a04a-66cb52138188)

![image](https://github.com/google/site-kit-wp/assets/94359491/5d16c5b6-01bf-4565-96f1-df7048b0ac92)

![image](https://github.com/google/site-kit-wp/assets/94359491/0501a6c8-dfe8-40c9-8c24-c4d941ec2fe9)

![image](https://github.com/google/site-kit-wp/assets/94359491/99c75926-4637-480f-8983-c7d390578b5d)

![image](https://github.com/google/site-kit-wp/assets/94359491/73d08e0a-7a5f-4d80-aa94-f68976d470bb)

![image](https://github.com/google/site-kit-wp/assets/94359491/c6ba9576-30a1-4b58-83f8-68942c3fd0e9)

![image](https://github.com/google/site-kit-wp/assets/94359491/e9284864-e16f-417b-a31d-04139fb2b4f7)

![image](https://github.com/google/site-kit-wp/assets/94359491/b3eb7ddc-7054-4101-81c5-079ae345e600)


**Dev**

![image](https://github.com/google/site-kit-wp/assets/94359491/30873144-8423-4fc2-9b07-cabcd37379aa)

![image](https://github.com/google/site-kit-wp/assets/94359491/2818d075-5ed1-4c0d-86c2-1e8a8ec683aa)

![image](https://github.com/google/site-kit-wp/assets/94359491/6ade5f87-a890-45db-a5e3-e7d6c0765e29)

![image](https://github.com/google/site-kit-wp/assets/94359491/56931ddf-ab2f-4c5e-bfee-1c89b8764ef2)

![image](https://github.com/google/site-kit-wp/assets/94359491/a585615c-103c-4ec1-a0ad-9aefaea98b62)

![image](https://github.com/google/site-kit-wp/assets/94359491/71d16f74-66ee-4438-b662-d5dd00be0228)

![image](https://github.com/google/site-kit-wp/assets/94359491/a39885cd-a7de-4ee2-a28e-e20c3853b129)

![image](https://github.com/google/site-kit-wp/assets/94359491/c27b622a-800b-472c-b5dd-2a90783f00f9)

</details>
